jax/_src/lax/lax.py

```diff
@@ -831,11 +831,15 @@ def _convert_to_hlo_attr(self, lhs_dtype: DTypeLike,
 
       if self in (DotAlgorithm.Preset.ANY_F8_ANY_F8_F32,
                   DotAlgorithm.Preset.ANY_F8_ANY_F8_F32_FAST_ACCUM):
-        fp8_dtypes = (np.dtype(dtypes.float8_e4m3b11fnuz),
+        fp8_dtypes = [np.dtype(dtypes.float8_e4m3b11fnuz),
```

Comment:
I would leave this as a tuple – it's unlikely to cause issues I think, but someone somewhere might be relying on this and I see no good reason for changing it

Reply:
The `fp8_dtypes` variable is a local variable used only on line 843. In this context, it doesn't significantly affect functionality whether it's a tuple or a list. However, for consistency with other parts of the code where we check type_name using `in` or `not in`, we typically use a list. Therefore, I believe changing it to a list would enhance uniformity across the codebase.
```py
 if lhs_dtype not in fp8_dtypes or rhs_dtype not in fp8_dtypes:
```